docs/running-on-kubernetes.md

```diff
@@ -140,6 +140,13 @@ namespace as that of the driver and executor pods. For example, to mount a secre
 --conf spark.kubernetes.executor.secrets.spark-secret=/etc/secrets
 ```
 
+To add a secret as an env variable to either the driver or the executor container use the following options to the
```

Comment:
s/`To add a secret as an ...` /`To use a secret through an environment variable`/.

Reply:
Ok.